src/zepto.js

```diff
@@ -437,9 +437,10 @@ var Zepto = (function() {
 
     ready: function(callback){
       // don't use "interactive" on IE <= 10 (it can fired premature)
+      // pipe any ready callback call over task/event queue (#1235)
```

Comment:
I would prefer to save 'link' to the PR, because it precludes wasting time on `git blame` for future investigations. I’m okay with deleting that comment and adding PR id to the previous comment (which about IE <= 10).